paper/analysis_memo.Rmd

```diff
@@ -81,18 +87,69 @@ The system support domain included 6 questions, each assessed with responses of
 
 The empathy domain, derived from the Toronto Empathy Questionnaire (TEQ), included 16 questions measured on a 5-point Likert scale for a total possible score of 64. Responses were scored on a scale where positively worded items (where “always” was the preferred answer) were scored as follows: never = 0; rarely = 1; sometimes = 2; often = 3; always = 4. Conversely, negatively worded items (where “never” was the desired response) had the scores reversed to account for the preferred response, with “never” obtaining the highest score of 4. Higher scores were indicative of greater self-reported empathic behavior.
 
-Composite variables for each domain will be created by summing the scores of each survey item within their respective domain. 
+Composite variables for each domain will be created by summing the scores of each survey item within their respective domain.
 
 The survey asked two additional questions on provider practices which will be analyzed and reported separately due to it not being applicable to all respondents. The first question asked participants if they had identified a female patient who was the victim of domestic violence within the past month. If yes, it asked how many patients. The second question used skip logic and was only applicable to those who answered “yes” to the first question. It asked respondents which actions they had taken after identifying a woman subjected to domestic violence. Questions were scored as 1 for correct (“yes”) responses and 0 for incorrect (“no” or “I don’t know”) responses. Higher scores were indicative of more appropriate actions taken to support women identified as victims of GBV.
 
 # Results
 
 ## Demographic information
+Matched data was obtained for 78 individuals, representing a total of 234 surveys out of the 676 conducted across the three timepoints. There were no significant demographic differences between individuals with less than 3 timepoints and all three timepoints, leading us to believe the smaller longitudinal sample remained representative of the intervention population. Demographic characteristics and work history of participants at the baseline are presented in Table 1. The overall sample was `r female$ratio`% female and `r male$ratio`% male. `r age$ratio`% of the sample fell within the 25-34 age group. The most prevalent occupational role was medical doctor, accounting for `r doctor$ratio`%, followed by nurses at `r nurse$ratio`%, and midwives at `r midwife$ratio`%. On average, participants had been practicing in their clinical roles for approximately `r signif(mean(filtered_data$position_years_clean), 2)` years. Only `r previous_training$ratio`% of participants had received any previous GBV training.
+
 ```{r}
 demographic_table %>%
   modify_caption("**Table 1. Participant Characteristics**")
+
+female <- filtered_data %>%
```

Comment:
What's all this for? Can't you just use the table one created in the table 1 script?

Reply:
I was struggling to be able to reference the actual percentage of females, males, etc. using inline code. I couldn't find a way to "call" that number from the table. My research online made it seem like I had to create an object of some sort to reference in the inline code. Is there an easier way?

Reply:
I also wasn't sure if I should include all this code in the table 1 script instead of here, assuming I even need it all!